tms-common/src/main/java/org/intalio/tempo/workflow/util/jpa/TaskFetcher.java

```diff
@@ -173,7 +174,7 @@ private Query buildQuery(Map parameters) {
 					baseQuery + taskClass.getSimpleName() + QUERY_GENERIC2_FOR_ADMIN);
 			}else{
```

Comment:
@bapiraju please run code formatter and rebase the two pull request.
@himanshu1587